api/controllers/v2/slack.ts

```diff
@@ -89,17 +89,28 @@ export const getDeployableJobs = async (
   const deployable = [];
 
   for (let i = 0; i < values.repo_option.length; i++) {
-    // e.g. mongodb/docs-realm/master => (site/repo/branch)
-    const [repoOwner, repoName, branchName] = values.repo_option[i].value.split('/');
+    let repoOwner: string, repoName: string, branchName: string, directory: string | undefined;
+    const splitValues = values.repo_option[i].value.split('/');
+
+    if (splitValues.length === 3) {
+      // e.g. mongodb/docs-realm/master => (owner/repo/branch)
+      [repoOwner, repoName, branchName] = splitValues;
+    } else if (splitValues.length === 4) {
```

Comment:
Is there a reason why we don't check for the Monorepo feature flag here like in v1? Is it because the enhanced Autobuilder isn't available in dotcomprd yet?

Reply:
You're right, I should add that!